special-interest-groups/sig-planner/roles-and-organization-management.md

```diff
@@ -0,0 +1,64 @@
+# Planner SIG Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Planner SIG is to have
+the **Active Contributor** community role in the planner/statistics package.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 3 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Planner SIG Reviewers, Committer or Maintainer.
+
+#### From `Active Contributor` to `Reviewer`
+
+Requirements:
+
+* Have at least 10 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Committers or 2 Maintainers.
+
+#### From `Reviewer` to `Committer`
+
+Requirements:
+
+* Have at least 30 PRs merged in the planner/statistics package within one year.
```

Comment:
how about 40 or 50?

Reply:
Add one more requirement that there must be enough valuable issues fixed by him/her.